modules/odc-viewing-application-topology.adoc

```diff
@@ -24,7 +19,16 @@ image::odc_application_topology.png[]
 
 The application resource name is appended with indicators for the different types of resource objects as follows:
 
-* *DC*: DeploymentConfigs
+* *CJ*: CronJob
 * *D*: Deployment
-* *SS*: StatefulSet
+* *DC*: DeploymentConfig
 * *DS*: Daemonset
+* *J*: Job
+* *P*: Pod
+* *SS*: StatefulSet
+* image:odc_serverless_app.png[title="Serverless Application"] (Knative): A serverless application.
++
+[NOTE]
+====
+Serverless applications take some time to load and display on the *Topology* view. When you create a serverless application, it first creates a service resource and then a revision. After that it is deployed and displayed on the Topology view. If it is the only workload, you might be redirected to the *Add* page. Once the revision is deployed, the serverless application is displayed on the *Topology* view.
```

Comment:
I'm not sure what type of UI element the Topology view is, but the styling should be consistent throughout. If it's a clickable element, like a tab, it should be bold. If it's just a page title, it should not be. (The discussion of GUI elements starts on page 206 of the IBM Style Guide.)
s/After that/After that,
If Add is a page instead of a clickable element, it shouldn't be in bold.
s/Once/After

Reply:
Topology view is a whole, how to put it, section in gui. Say, "issues" in github. A separate page within ocp gui. 

@kalexand-rh thanks for the other remarks. fixed.

Reply:
*Add* is a clickable element, but it also represents a page. Leaving as is.

otherwise fixed, thanks

Reply:
Thanks @pmacko1!